freqtrade/optimize/optimize_reports.py

```diff
@@ -496,6 +531,8 @@ def text_table_add_metrics(strat_results: Dict) -> str:
     if len(strat_results['trades']) > 0:
         best_trade = max(strat_results['trades'], key=lambda x: x['profit_ratio'])
         worst_trade = min(strat_results['trades'], key=lambda x: x['profit_ratio'])
+        zero_duration_trades_percent =\
+            100.0 / strat_results['total_trades'] * strat_results['zero_duration_trades']
```

Comment:
This ... fails me with key-error when using `hyperopt-show` on a hyperopt file not done with this version.

I'm not 100% convinced if we do have to have this kind of backward compatibility though - as that might complicate this whole file - or at least everything in this method as we can't rely on anything "new" to be there.